server.js

```diff
@@ -0,0 +1,21 @@
+const Hapi = require('hapi');
+const Hoek = require('hoek');
+
+const log = require('./logger.js');
+
+Hoek.assert(process.env.API_HOST, 'Must define API_HOST');
+Hoek.assert(process.env.PORT, 'Must define PORT');
+
+var connection = {
+  host: process.env.API_HOST,
+  port: process.env.PORT
+};
+
+const server = new Hapi.Server();
+server.connection(connection); 
+
+server.register(require('./api'), function(err) {
```

Comment:
you might also want to register the logger as a plugin so that we capture server logged events:
```javascript
register: hapiBunyan,
options: {
      logger: log
}
```